nightmare/helpers/connectionsDSL.js

```diff
@@ -1,216 +1,302 @@
-/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
+/* eslint-disable max-lines */
+// eslint-disable-next-line import/no-extraneous-dependencies
 import Nightmare from 'nightmare';
 import selectors from './selectors.js';
 
 Nightmare.action('connections', {
   waitForRelationHubs(done) {
-    this.wait('.relationshipsHub').then(() => { done(); });
+    this.wait('.relationshipsHub').then(() => {
+      done();
+    });
   },
   sortBy(orderByText, done) {
     this.waitToClick(selectors.connections.sortMenu)
-    .evaluate((sortBy) => {
-      const helpers = document.__helpers;
-      const sortOptions = helpers.querySelectorAll('.sort-buttons .Dropdown-option a');
+      .evaluate(sortBy => {
+        const helpers = document.__helpers;
+        const sortOptions = helpers.querySelectorAll(
+          '.sort-buttons > div.Dropdown.order-by.is-active .Dropdown-option a'
```

Comment:
This is the only "real" change to the document, everything else is just Prettier doing its thing.